src/app/cli/interpreter.cpp

```diff
@@ -1007,13 +1006,27 @@ Interpreter::Value Interpreter::ProcessStartJob(CommissionerAppPtr &aCommissione
     Error       error;
     uint16_t    port;
     std::string existingCommissionerId;
+    bool        connectOnly = false;
 
     VerifyOrExit(aExpr.size() >= 3, error = ERROR_INVALID_ARGS(SYNTAX_FEW_ARGS));
     SuccessOrExit(error = ParseInteger(port, aExpr[2]));
-    SuccessOrExit(error = aCommissioner->Start(existingCommissionerId, aExpr[1], port));
+
+    {
+        auto it = std::find(mContext.mCommandKeys.begin(), mContext.mCommandKeys.end(), "--connect");
+
+        if (it != mContext.mCommandKeys.end())
+        {
+            connectOnly = true;
+            error       = aCommissioner->Connect(aExpr[1], port);
```

Comment:
It seems more straightforward to add a new "connect" command rather than the optional "--connect", given the fact that both the commissioner library and CommissionerApp have the "Connect()" API

Reply:
It was the initial thought. Changed to this way considering the visually matching start/stop command pairs.

cc @bukepo for vis.